Classes/ScrnAA12Fire.uc

```diff
@@ -3,4 +3,5 @@ class ScrnAA12Fire extends AA12Fire;
 defaultproperties
 {
      ProjectileClass=Class'ScrnBalanceSrv.ScrnAA12Bullet'
+     Spread=900.0//1125.000000
```

Comment:
Nope. Sorry. I don't want AA12 to become even more OP.

Reply:
I think the AA12 is underpowered behaves similarly to the Nailgun but costs 4 times as much, without considering penetration it also has almost identical damage output (1175 vs 1102), slightly more damage for total ammo (28200 vs 27468), and only twice the damage per mag (4700 vs 2646)
Versus other support weapons in ammo efficiency, the Nailgun offers the ability to single fire nails for better accuracy and ammo efficiency on small zeds, bouncing nails in tight spaces like bioticslab lets you more zeds than the spread allows or even hit the same zed twice.
AA12 only saves ammo by splitting 28200 into 120 shots versus 23870 in 62 shots for the pump/combat shotgun, or 32400 in 90 shots for KSG. 
Damage per minute versus other semiauto shotguns, the AA12 has 28.20k vs nailgun's 27.34k, or combat shotgun 23.87k, with tactical reload you skip 30% of the animation which brings the damage per minute output to around 40k, which matches the Saiga from ScrN Weapon Pack
With regards to penetration the nailgun has 75% on small zeds, and the KSG, Combat Shotgun and AA12 have 90%, and the Saiga has 70%, I need to test this more because I don't get to line up that many zeds often.
I think the OP shotgun is actually the Saiga, it already has tactical reload with similar times to the new AA12 (2.15 down from 3.13), 5040 damage per mag( or 5670 after tactical reload), and total damage of 45360, twice more than the AA12 and a little less than the hunting shotgun (48980). And it does it all in 8 weight blocks and with 835 spread for the cost of 2.5 nailguns. Taking the nailgun as the base measure of a magazine reload semiauto shotgun, I have to give up 2 weight blocks and 600 dosh to use the Saiga, but for the AA12 I need to give up 4 weight blocks and 1200 just to get half the damage for total ammo in return for better penetration by about 28%. 
Though counting headshots, the AA12 does have a 1.65x headshot multiplier that other shotguns also have, if you hit all pellets to the head that brings the damage per second up to 1938, by comparison the combat shotgun (1.65x) will do 3176 damage for all pellet headshots in 1 second, and the Saiga (1.1x) will do 1155, at the moment the saiga has the lowest spread, so I am proposing the change to also lower the AA12 spread as well to get as many headshots at range as the saiga does. 
I'll test it some more but I still think the AA12 isn't good enough for how much it weighs and costs, given its stats it still seems best for shooting groups of small zeds and getting some headshot damage on charging large zeds before switching to the hunting shotgun.

Reply:
Wow, that is a great balance feedback. Thank you! I accept your point regarding AA12 vs Nailgun comparison. But instead of lowering AA12's spread I'd suggest raising Nailgun's (1250, twice lower than vanilla, only +11% to default shotgun's 1125). Vanilla's 2500 spread makes Nailgun totally suck. But ScrN 1250 seems too OP, especially after ProjPerFire back to 7 (initially ScrN lowered it down to 5 together with Spread reduction). I will try raising Nailgun's spread to 1500-2000.

Returning back to AA12, its bigger pros is DPS, not DPM or total damage. When you pull the AA12 trigger, everything in the hallway dies. Everything but big zeds, but using AA12 vs big ones is considered aim. There are combos, there are trickshots. They do not involve AA12. AA12 is an ultimate hallway cleaner. Tactical reload will allow more efficient usage of single shots and short bursts. And I accept that. But lowering spread doesn't sound right to me.

Anyway, remove the spread line from the current pull request. We can continue the discussion on ScrN forums or you can create a separate pull request for that.

Reply:
I made a thread for the nailgun balance changes. I also think spread shouldn't be increased or else using it in hallways will result in the zeds being knocked in more random horizontal directions.